tests/write_netex_france.rs

```diff
@@ -29,14 +27,36 @@ fn test_write_netex_france() {
             get_test_datetime(),
         );
         netex_france_exporter.write(output_dir).unwrap();
-        compare_output_dir_with_expected_content(&output_dir, None, "tests/fixtures/netex_france");
+        compare_output_dir_with_expected_content(
+            &output_dir,
+            None,
+            "tests/fixtures/netex_france/output_netex_fr",
+        );
     });
 }
 
+#[test]
+fn test_write_netex_france_from_ntfs() {
+    let model = transit_model::ntfs::read("tests/fixtures/netex_france/input_ntfs").unwrap();
+    test_write_netex_france(model);
+}
+
+#[test]
+fn test_write_netex_france_from_gtfs() {
+    let model = transit_model::gtfs::read_from_path(
+        "tests/fixtures/netex_france/input_gtfs",
+        None,
+        None,
+        false,
+    )
+    .unwrap();
+    test_write_netex_france(model);
+}
+
 #[test]
 #[cfg(feature = "xmllint")]
 fn validate_xml_schemas() {
-    let paths = fs::read_dir("tests/fixtures/netex_france/")
+    let paths = fs::read_dir("tests/fixtures/netex_france/output_netex_fr/")
```

Comment:
Since the folder is already inside a `netex_france`, maybe you can just a structure of folders like `netex_france/output/`.